README.md

```diff
@@ -10,6 +10,8 @@ To get started with a library, see the README.md file located in the library's p
 
 For tutorials, samples, quick starts, and other documentation, go to [Azure for .NET Developers](https://docs.microsoft.com/dotnet/azure/).
 
+For a list of changes made between releases, refer to the [change log](sdk/tables/Azure.Data.Tables/CHANGELOG.md).
```

Comment:
Did you mean to link to the `Tables` changelog from the main repository ReadMe?

Reply:
I did not. I meant to link the Tables changelog to the Tables readme. Good catch! I will redo and resubmit.